web/controllers/tokenController/index.ts

```diff
@@ -178,7 +182,11 @@ export async function getNewTokenHandler(req: Request, res: Response) {
             }
 
             if (person !== undefined) {
-                if (allowedToRequestToken(person)) {
+                if(person.verifiedAt == null) {
+                    status = 409;
```

Comment:
What happens in the frontend if the api call returns `409`? The user should be encouraged to check his mails and a possible error message may be misleading except it states that a verification token is sent.

Reply:
The frontend catches this error and shows an error message saying that the user should check their emails

Reply:
Alright, sounds good👍